server/src/test/java/com/broadcom/lsp/cobol/usecases/TestParagraphNotDefined.java

```diff
@@ -0,0 +1,41 @@
+package com.broadcom.lsp.cobol.usecases;
+
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.Position;
+import org.eclipse.lsp4j.Range;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TestParagraphNotDefined extends NegativeUseCase {
+    private static final String TEXT =
+            "        IDENTIFICATION DIVISION.\r\n"
+                    + "        PROGRAM-ID. test1.\r\n"
+                    + "        DATA DIVISION.\r\n"
+                    + "        WORKING-STORAGE SECTION.\r\n"
+                    + "        PROCEDURE DIVISION.\r\n"
+                    + "        PROGA.\r\n"
+                    + "           PERFORM PRINT-DATA.";
```

Comment:
I think that you can wrap PRINT-DATA like:
```
+ "           PERFORM {#PRINT-DATA}.";
```
It will calculate the position automatically. Please, refer to `UseCaseEngine` docs for details. I didn't have a chance to use it but want to try someday. So, my suggestion can be wrong.

Reply:
In that case there will be 2 diagnostic errors. One of them will be that counts for usages and declarations do not match. To avoid this I did not use `{#PRINT-DATA}`

Reply:
Fixed with Andrei's help :-)